gdpr-what-it-means-for-readthedocs.rst

```diff
@@ -0,0 +1,79 @@
+.. post:: May 25, 2018
+   :tags: privacy, gdpr, advertising
+   :author: David
+   :location: SAN
+
+GDPR: What it means for Read the Docs
+=====================================
+
+Your email inbox has probably been bombarded
+over the last few days and weeks with "Updates to our Privacy Policy".
+These emails pertain to an EU law called the General Data Protection Regulation (GDPR)
+which comes into effect today.
+
+The goal of the GDPR is to put users back in control of their data.
+It is an important step toward respecting users' privacy.
+The days of collecting as much data on as many people as possible
+without consent and sharing it with anyone willing to pay for it are over.
```

Comment:
This is a bit optimistic :)

Reply:
Definitely. It will change some people (mostly companies with something to lose) and others will disregard it as too bureaucratic. I wouldn't mind adding a bit of humor to the post though. Perhaps something like:

    Maybe that is a bit optimistic but it has helped us build our privacy infrastructure and it does seem many companies are beginning to take privacy seriously at least if measured by the volume of privacy-related email received.

Reply:
Eh, this is a bit repetitive with the first paragraph. I'll just publish this as-is.